src/includes/dinit-env.h

```diff
@@ -13,8 +13,9 @@ extern environment main_env;
 // Read an environment file and set variables in the current environment.
 //   file - the file to read
 //   log_warnings - if true, syntactic errors are logged
+//   throw_on_open_failure - if true, failure in reading file will result in a std::system_error exception
```

Comment:
The comment is incorrect. If true, failure to _open_ the file will result in a `std::system_error` exception, as the name suggests. I.e. the comment should say "failure to open" not "failure in reading".

(Failure in _reading_ the file, once open, always will result in a `std::ios::failure` exception, regardless of the parameter).